torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -150,6 +150,7 @@ def speculate_subgraph(
     graph_checkpoint,
     checkpoint,
     description,
+    source_target,
```

Comment:
can we put this after the `*`? Also, what is the type of this? (HigherOrderOpVariable ?)

Reply:
Sure! It's the .value of HigherOrderOpVariable, which is the target of the proxy that we created for the higherOrderOperator.

Reply:
Oh, so this is just a string. The name "source_target" is not that descriptive, could we add a comment with what you said?

Reply:
Sounds good!